scripts/build

```diff
@@ -57,6 +77,35 @@ curl ${longhorn_image_url}>>${image_list_file}
 # get k3s image list
 curl ${K3S_IMAGE_URL}>>${image_list_file}
 
+# get rancher image list
+RANCHER_REMOTE_IMAGES=rancher-remote-images.txt
+RANCHER_IMAGES=scripts/images/rancher-images.txt
+
+if [ ! -f "$RANCHER_REMOTE_IMAGES" ]; then
+    touch $RANCHER_REMOTE_IMAGES
+fi
+
+curl -fL -o "${RANCHER_REMOTE_IMAGES}" "${RANCHER_IMAGE_URL}"
+
+while IFS= read -r img || [ -n "$img" ]
+do
+  while IFS= read -r img_rq || [ -n "$img_rq" ]
+  do
+    if [[ $img == "$img_rq"* ]]; then
+      echo "$img" >> ${image_list_file}
+    fi
+  done < "$RANCHER_IMAGES"
+done < "$RANCHER_REMOTE_IMAGES"
+echo "Done adding rancher images"
+
+# add cert-manager image list
+CERT_MANAGER_IMAGES=scripts/images/cert-manager-images.txt
```

Comment:
Can we generate the image list with $CERT_MANAGER_VERSION so we don't need to update both the script and the image files when we bump cert-manager version?